list.go

```diff
@@ -0,0 +1,85 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/tabwriter"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+var listCommand = cli.Command{
+	Name:  "list",
+	Usage: "lists containers started by runc with the given root",
+	Action: func(context *cli.Context) {
+
+		// preload the container factory
+		if factory == nil {
+			err := factoryPreload(context)
+			if err != nil {
+				logrus.Fatal(err)
+				return
+			}
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
+		absRoot, err := filepath.Abs(root)
+		if err != nil {
+			logrus.Fatal(err)
+			return
+		}
+		list, err := ioutil.ReadDir(absRoot)
+
+		w := tabwriter.NewWriter(os.Stdout, 12, 1, 3, ' ', 0)
+		fmt.Fprint(w, "ID\tSTARTED\tSTATUS\tPID\tROOTFS\n")
```

Comment:
I prefer the order to be: "ID\tPID\tSTATUS\tSTARTED\tROOTFS\n", it's more consistent with linux ps output, and maybe list BUNDLE instead of ROOTFS, WDYT?